patches/chrome-android-BUILD.gn.patch

```diff
@@ -19,6 +19,15 @@ index 0c5467af2bce341e0b3f0835899c820f60adbc87..ad26d5d307b168283b9d0bab7c9d4f1d
      "//chrome/android/webapk/libs/common:splash_resources",
      "//chrome/app:java_strings_grd",
      "//components/autofill/android:autofill_java_resources",
+@@ -160,7 +161,7 @@ android_resources("chrome_download_java_resources") {
+ 
+ java_strings_grd("chrome_strings_grd") {
+   defines = chrome_grit_defines
+-  grd_file = "java/strings/android_chrome_strings.grd"
++  grd_file = "java/strings/brave_android_chrome_strings.grd"
```

Comment:
I think there is no way to avoid this change for replacing `android_chrome_strings.grd` with `brave_android_chrome_strings.grd`.
and just `brave_` is prepended because we already use `android_brave_strings.grd` for our own strings.